tests/net/mod.rs

```diff
@@ -0,0 +1,95 @@
+use std::env;
+use std::net::{Ipv4Addr, SocketAddr};
+use std::sync::atomic::{AtomicU16, Ordering};
+
+use drop::crypto::key::exchange::Exchanger;
+use drop::net::{
+    Connector, DirectoryConnector, DirectoryListener, DirectoryServer,
+    Listener, TcpConnector, TcpListener,
+};
+
+use tokio::task;
+
+use tracing::trace_span;
+use tracing_futures::Instrument;
+use tracing_subscriber::FmtSubscriber;
+
+/// Initialize an asynchronous logger for test environment
+pub fn init_logger() {
```

Comment:
duplicated from src/test/log.rs

Reply:
These are required we can't actually access the drop's internals from an integration test